packages/discord.js-utilities/README.md

```diff
@@ -0,0 +1,83 @@
+<div align="center">
+
+![Sapphire Logo](https://cdn.skyra.pw/gh-assets/sapphire.png)
+
+# @sapphire/discord.js-utilities
+
+**Discord.js specific utilities for your JavaScript/TypeScript bots.**
+
+[![GitHub](https://img.shields.io/github/license/sapphire-project/utilities)](https://github.com/sapphire-project/utilities/blob/main/LICENSE.md)
+[![Total alerts](https://img.shields.io/lgtm/alerts/g/sapphire-project/utilities.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/sapphire-project/utilities/alerts/)
+[![Language grade: JavaScript](https://img.shields.io/lgtm/grade/javascript/g/sapphire-project/utilities.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/sapphire-project/utilities/context:javascript)
+[![Coverage Status](https://coveralls.io/repos/github/sapphire-project/utilities/badge.svg?branch=main)](https://coveralls.io/github/sapphire-project/utilities?branch=main)
+[![npm bundle size](https://img.shields.io/bundlephobia/min/@sapphire/discord.js-utilities?logo=webpack&style=flat-square)](https://bundlephobia.com/result?p=@sapphire/discord.js-utilities)
+[![npm](https://img.shields.io/npm/v/@sapphire/discord.js-utilities?color=crimson&logo=npm&style=flat-square)](https://www.npmjs.com/package/@sapphire/discord.js-utilities)
+[![Depfu](https://badges.depfu.com/badges/ec42ff3d6bae55eee1de4749960852b3/count.svg)](https://depfu.com/github/sapphire-project/utilities?project_id=15195)
+
+</div>
+
+## Description
+
+When developing Discord bots or frameworks for Discord bots we often run into certain constants such as regex's, channel type resolutions and other common functions that would be really nice to get from 1 centralized place. That's where @sapphire/discord.js-utilities come in. This package provides common Discord specific utilities for your Discord bots.
```

Comment:
```suggestion
There many functions that are always extremely useful to have when developing bots with [Discord.JS]. Such as channel type guards, paged messages and more. For this purpose we provide @sapphire/discord.js-utilities. This is also an extension of sorts on our package [@sapphire/discord-utilities], which is agnostic to the API library you're using for your bot. That is to say, @sapphire/discord.js-utilities re-exports all functions from [@sapphire/discord-utilities]
```

And add to the bottom of the readme:
```md
[Discord.JS]: https://discord.js.org
[@sapphire/discord-utilities]: ../discord-utilities
```

(gotta check if that last one works but can only check once you're committed it to your branch)